docs/source/user_guide/installation/index.rst

```diff
@@ -45,6 +45,10 @@ Dependencies
 ------------
 .. _install.required_dependencies:
 
+.. warning::
+
+    The list of dependencies below may be outdated. Please refer to the official repository or package manager for the most up-to-date versions.
+
```

Comment:
```suggestion
    The list of dependencies below might be outdated. Please refer to the pyproject.toml file to find all dependencies.

```

Reply:
This should be more accurate.

Reply:
I have corrected it in the latest commit